Put examples into right place and gschem imported symbols available

Index: xschem_library/Makefile
--- xschem_library/Makefile.orig
+++ xschem_library/Makefile
@@ -5,47 +5,49 @@ all:
 include ../Makefile.conf
 
 install: FORCE
-	$(SCCBOX) mkdir -p "$(system_library_dir)" "$(XDOCDIR)"/examples "$(XDOCDIR)"/pcb
-	$(SCCBOX) mkdir -p "$(XDOCDIR)"/logic "$(XDOCDIR)"/xTAG "$(XDOCDIR)"/binto7seg 
-	$(SCCBOX) mkdir -p "$(XDOCDIR)"/symgen "$(XDOCDIR)"/ngspice "$(XDOCDIR)"/rulz-r8c33
-	$(SCCBOX) mkdir -p "$(XDOCDIR)"/rom8k "$(XDOCDIR)"/gschem_import/sym
+	$(SCCBOX) mkdir -p "$(system_library_dir)"
+	$(SCCBOX) mkdir -p "$(system_library_dir)"/../gschem_import
+	$(SCCBOX) mkdir -p "$(XEXAMPLESDIR)"/examples "$(XEXAMPLESDIR)"/pcb
+	$(SCCBOX) mkdir -p "$(XEXAMPLESDIR)"/logic "$(XEXAMPLESDIR)"/xTAG "$(XEXAMPLESDIR)"/binto7seg 
+	$(SCCBOX) mkdir -p "$(XEXAMPLESDIR)"/symgen "$(XEXAMPLESDIR)"/ngspice "$(XEXAMPLESDIR)"/rulz-r8c33
+	$(SCCBOX) mkdir -p "$(XEXAMPLESDIR)"/rom8k "$(XEXAMPLESDIR)"/gschem_import/sym
 	$(SCCBOX) install -f -d devices/*.sym "$(system_library_dir)"
 	$(SCCBOX) install -f -d devices/*.sch "$(system_library_dir)"
-	$(SCCBOX) install -f -d examples/*.sym "$(XDOCDIR)"/examples
-	$(SCCBOX) install -f -d examples/*.sch "$(XDOCDIR)"/examples
-	$(SCCBOX) install -f -d examples/stimuli.* "$(XDOCDIR)"/examples
-	$(SCCBOX) install -f -d binto7seg/*.sym "$(XDOCDIR)"/binto7seg
-	$(SCCBOX) install -f -d binto7seg/*.sch "$(XDOCDIR)"/binto7seg
-	$(SCCBOX) install -f -d xTAG/*.sym* "$(XDOCDIR)"/xTAG
-	$(SCCBOX) install -f -d xTAG/*.sch "$(XDOCDIR)"/xTAG
-	$(SCCBOX) install -f -d logic/*.sym "$(XDOCDIR)"/logic
-	$(SCCBOX) install -f -d logic/*.sch "$(XDOCDIR)"/logic
-	$(SCCBOX) install -f -d logic/stimuli.* "$(XDOCDIR)"/logic
-	$(SCCBOX) install -f -d rulz-r8c33/*.sch "$(XDOCDIR)"/rulz-r8c33
-	$(SCCBOX) install -f -d rulz-r8c33/*.sym* "$(XDOCDIR)"/rulz-r8c33
-	$(SCCBOX) install -f -d pcb/*.sch "$(XDOCDIR)"/pcb
-	$(SCCBOX) install -f -d pcb/*.sym "$(XDOCDIR)"/pcb
-	$(SCCBOX) install -f -d ngspice/*.sym "$(XDOCDIR)"/ngspice
-	$(SCCBOX) install -f -d ngspice/*.sch "$(XDOCDIR)"/ngspice
-	$(SCCBOX) install -f -d ngspice/stimuli.* "$(XDOCDIR)"/ngspice
-	$(SCCBOX) install -f -d rom8k/*.sym "$(XDOCDIR)"/rom8k
-	$(SCCBOX) install -f -d rom8k/*.sch "$(XDOCDIR)"/rom8k
-	$(SCCBOX) install -f -d xschem_simulator/*.sym "$(XDOCDIR)"/xschem_simulator
-	$(SCCBOX) install -f -d xschem_simulator/*.sch "$(XDOCDIR)"/xschem_simulator
-	$(SCCBOX) install -f -d rom8k/stimuli.* "$(XDOCDIR)"/rom8k
-	$(SCCBOX) install -f -d symgen/*.sym* "$(XDOCDIR)"/symgen
-	$(SCCBOX) install -f -d gschem_import/*.sym "$(XDOCDIR)"/gschem_import
-	$(SCCBOX) install -f -d gschem_import/*.sch "$(XDOCDIR)"/gschem_import
-	$(SCCBOX) install -f -d gschem_import/sym/*.sym "$(XDOCDIR)"/gschem_import/sym
+	$(SCCBOX) install -f -d examples/*.sym "$(XEXAMPLESDIR)"/examples
+	$(SCCBOX) install -f -d examples/*.sch "$(XEXAMPLESDIR)"/examples
+	$(SCCBOX) install -f -d examples/stimuli.* "$(XEXAMPLESDIR)"/examples
+	$(SCCBOX) install -f -d binto7seg/*.sym "$(XEXAMPLESDIR)"/binto7seg
+	$(SCCBOX) install -f -d binto7seg/*.sch "$(XEXAMPLESDIR)"/binto7seg
+	$(SCCBOX) install -f -d xTAG/*.sym* "$(XEXAMPLESDIR)"/xTAG
+	$(SCCBOX) install -f -d xTAG/*.sch "$(XEXAMPLESDIR)"/xTAG
+	$(SCCBOX) install -f -d logic/*.sym "$(XEXAMPLESDIR)"/logic
+	$(SCCBOX) install -f -d logic/*.sch "$(XEXAMPLESDIR)"/logic
+	$(SCCBOX) install -f -d logic/stimuli.* "$(XEXAMPLESDIR)"/logic
+	$(SCCBOX) install -f -d rulz-r8c33/*.sch "$(XEXAMPLESDIR)"/rulz-r8c33
+	$(SCCBOX) install -f -d rulz-r8c33/*.sym* "$(XEXAMPLESDIR)"/rulz-r8c33
+	$(SCCBOX) install -f -d pcb/*.sch "$(XEXAMPLESDIR)"/pcb
+	$(SCCBOX) install -f -d pcb/*.sym "$(XEXAMPLESDIR)"/pcb
+	$(SCCBOX) install -f -d ngspice/*.sym "$(XEXAMPLESDIR)"/ngspice
+	$(SCCBOX) install -f -d ngspice/*.sch "$(XEXAMPLESDIR)"/ngspice
+	$(SCCBOX) install -f -d ngspice/stimuli.* "$(XEXAMPLESDIR)"/ngspice
+	$(SCCBOX) install -f -d rom8k/*.sym "$(XEXAMPLESDIR)"/rom8k
+	$(SCCBOX) install -f -d rom8k/*.sch "$(XEXAMPLESDIR)"/rom8k
+	$(SCCBOX) install -f -d xschem_simulator/*.sym "$(XEXAMPLESDIR)"/xschem_simulator
+	$(SCCBOX) install -f -d xschem_simulator/*.sch "$(XEXAMPLESDIR)"/xschem_simulator
+	$(SCCBOX) install -f -d rom8k/stimuli.* "$(XEXAMPLESDIR)"/rom8k
+	$(SCCBOX) install -f -d symgen/*.sym* "$(XEXAMPLESDIR)"/symgen
+	$(SCCBOX) install -f -d gschem_import/*.sym "$(XEXAMPLESDIR)"/gschem_import
+	$(SCCBOX) install -f -d gschem_import/*.sch "$(XEXAMPLESDIR)"/gschem_import
+	$(SCCBOX) install -f -d gschem_import/sym/*.sym "$(system_library_dir)"/../gschem_import
 
 uninstall: FORCE
-	$(SCCBOX) rm -f "$(system_library_dir)"/* "$(XDOCDIR)"/examples/* "$(XDOCDIR)"/pcb/* \
-                        "$(XDOCDIR)"/ngspice/* "$(XDOCDIR)"/symgen/* "$(XDOCDIR)"/logic/* "$(XDOCDIR)"/xTAG/* \
-                        "$(XDOCDIR)"/rom8k/* "$(XDOCDIR)"/xschem_simulator/ "$(XDOCDIR)"/binto7seg/* "$(XDOCDIR)"/rulz-r8c33/* \
-                        "$(XDOCDIR)"/gschem_import/*.sym \
-                        "$(XDOCDIR)"/gschem_import/*.sch \
-                        "$(XDOCDIR)"/gschem_import/sym/*.sym \
-                        "$(XDOCDIR)"/gschem_import/sym
+	$(SCCBOX) rm -f "$(system_library_dir)"/* "$(XEXAMPLESDIR)"/examples/* "$(XEXAMPLESDIR)"/pcb/* \
+                        "$(XEXAMPLESDIR)"/ngspice/* "$(XEXAMPLESDIR)"/symgen/* "$(XEXAMPLESDIR)"/logic/* "$(XEXAMPLESDIR)"/xTAG/* \
+                        "$(XEXAMPLESDIR)"/rom8k/* "$(XEXAMPLESDIR)"/xschem_simulator/ "$(XEXAMPLESDIR)"/binto7seg/* "$(XEXAMPLESDIR)"/rulz-r8c33/* \
+                        "$(XEXAMPLESDIR)"/gschem_import/*.sym \
+                        "$(XEXAMPLESDIR)"/gschem_import/*.sch \
+                        "$(XEXAMPLESDIR)"/gschem_import/sym/*.sym \
+                        "$(XEXAMPLESDIR)"/gschem_import/sym
 clean:
 
 distclean:
